pydocstyle.cfg

```diff
@@ -0,0 +1,15 @@
+[pydocstyle]
+# D100 Missing docstring in public module
+# D102	Missing docstring in public method
+# D103	Missing docstring in public function
```

Comment:
This actually here, because at the moment our code base actually contains a lot of those. This line can easily be removed when it is less noisy.